__generated/rr/rrweb/rr.js

```diff
@@ -4648,7 +4648,7 @@ var CanvasManager = class {
         return hadLoadingTime;
       };
       const promises = [];
-      promises.push(...getCanvas(timestamp).filter(filterElementStartTime).map((canvas) => this.snapshot(canvas)));
+      promises.push(...getCanvas(timestamp).filter(filterElementStartTime).map(this.snapshot));
```

Comment:
wondering why this generated rrweb change made it in? looks trivial tho

Reply:
Yeah, my bad... I can back this out before merging!